marimo/_runtime/packages/module_name_to_pypi_name.py

```diff
@@ -533,6 +533,7 @@ def module_name_to_pypi_name() -> dict[str, str]:
         "markdown": "Markdown",
         "marks": "pytest-marks",
         "markupsafe": "MarkupSafe",
+        "matplotlib-pyplot": "matplotlib",
```

Comment:
What's this for? `import matplotlib.pyplot as plt` already correctly prompts to install `matplotlib`. 

<img width="1357" height="568" alt="Image" src="https://github.com/user-attachments/assets/043364d8-4e68-40de-9906-975916c1b8cf" />